node/12/Dockerfile

```diff
@@ -1,6 +1,14 @@
-FROM node:12-slim
+FROM debian:buster-slim
 MAINTAINER Damien Carcel <damien.carcel@akeneo.com>
 
+RUN apt-get update && \
+    apt-get --yes install \
+        yarnpkg \
+        nodejs && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    ln -s /usr/bin/yarnpkg /usr/bin/yarn
```

Comment:
Needed to still have access to `yarn` command

Reply:
yarnpkg version is a vey old version. 
To use the recent yarn version we can install il from the npm install
An example here: https://github.com/akeneo/cloud-paas/pull/984/files

Reply:
Why?

Reply:
The real issue here is: what do we tell our integrators to do in our PIM documentation? If we tell them to use the `yarnpkg` version like JM did, then it is fine, regardless of how old the version provided is.
However, if we tell them, for instance, to use the official Yarn repo for Debian, which provided an up-to-date version, then we should do it.

Reply:
https://docs.akeneo.com/master/install_pim/manual/system_requirements/manual_system_installation_debian10.html#node

Since v4, we tell our partners and users to use the package provided by Debian.

Reply:
:ok_hand: 
Then we should do it everywhere:
- stop using the official `node` Docker image at all as a base, as JM propose,
- install the right Node + npm + Yarn version for our `akeneo/node:10` and `akeneo/node:12`.

Reply:
Another possibility is to say that Docker is the default and only supported way. Then
- find a way to not have a need for our custom `akeneo/node` images,
- then only use `node:xx` instead everywhere.